src/DotNetProjectFile.Analyzers/Analyzers/Generic/OnlyUseUTF8WithoutBom.cs

```diff
@@ -32,7 +32,7 @@ protected override void Register(ProjectFileAnalysisContext<ProjectTextFile> con
     {
         if (!context.File.Path.Exists) { return; }
 
-        using var reader = context.File.Path.OpenRead();
+        using var reader = context.File.Path.TryOpenRead();
```

Comment:
This does not lead to a crash when trying to Read on a null stream?

Reply:
The test proofs it not to fail. (That is the whole point of the `Stream.Null`, it is an empty readable stream).